### PR TITLE
Ready for release

### PR DESCRIPTION
**Pull request by Daron976**
### In this PR:
- We set up an HTML document with the relevant placeholders for data to be fed in from JS.
- We made a request to an API for the data.
- We created a popup window that comes to life upon a clicked button.
- We implemented counters for the data displayed on the homepage as well as the number of comments available.
- WE implemented a like functionality that updates the number of likes when a button is clicked.
- this is a link to the video [presentation](https://drive.google.com/file/d/1qFSTBIyO0a2sZHcDqIwErUd_5fVrlEFo/view?usp=sharing)

### CODE REVIEW

File: src/API keys [hunk 1]
```diff
@@ -0,0 +1 @@
+Involvement API ID: HXa9GIOD94bm1hT63nt8;
```
Comment (codepantha) [inline src/API keys:1]: - [ ] Confidential information about your API should not be stored on GitHub as it is a security risk. Please remove it 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi team 👋, 

I can tell that you have put in a lot of effort into this project so far 👏!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitFlow
✔️ No linter errors
✔️ Good UI design

## Required Changes ♻️
_Check the comments under the review._

- [ ] I appreciate your presentation skills 👏 However, according to project requirements, you and your coding partner should appear in the video and talk about the project.
- [ ] Please ensure that the like counter function is covered with unit tests using [the Jest Framwework](https://jestjs.io/) 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/API keys (hunk 1 above)
Comment (Daron976) [reply src/API keys:1]: @codepantha Please note that my partner was not available at the time of the video recording.
the point of the video was to present the code which I have already completed.

Comment (codepantha) [reply src/API keys:1]: I understand, but it is mandatory that you both should appear in the video presentation. I believe you still have time to pick an appropriate time when you both can be available to make the presentation.

**Comment by Iptysam** [comment]
The presentation video in the description has been updated

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hi there 👋🏽 , 

Great work on making the changes requested by a previous reviewer 👏 
To highlight 👍🏽 
- There are no linter errors
- All tests pass
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
- [ ] Like the last reviewer suggested, kindly ensure that both coding partners appear in the video presentation and talk about the project together. Also, apart from giving a description of the project, the requirements state that you highlight a piece of the code that you find interesting or are proud of making.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Daron976** [comment]
Please note that the PR has been updated with the new video as per request of the code reviewers

**Comment by Iptysam** [comment]
Presentation video updated in the Description

**Review by levy002 — APPROVED** [review]
### Hello @Daron976   & @Iptysam 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly make sure next time the video presentation must not go over 5 minutes maximum. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**